scripts/tls_client_auth/certificateauth.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -eu
+
```

Comment:
Suggest that a 'which openssl' here to check that the openssl command is avilable with a suitable message would be good...

(and for amy other commands that are used...)

Reply:
I believe that `type` is preferred to `which`, so added that